src/index.d.ts

```diff
@@ -0,0 +1,16 @@
+import { ReactNode } from "react";
```

Comment:
I think that this file should be `Image.d.ts` and `index.d.ts` should re-export that. Just so that the typings are symmetrical to the JS code. What do you think?